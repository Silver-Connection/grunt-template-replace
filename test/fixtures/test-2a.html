<h1>File A</h1>