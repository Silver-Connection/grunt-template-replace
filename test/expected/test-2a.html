<!doctype html>
<html lang="en-EN">
<head>
  <meta charset="utf-8">
  <title>Sample Page</title>
</head>
<body>

    <h1>File A</h1>

</body>
</html>